src/PixelStacker/Logic/Extensions/Extend.Color.cs

```diff
@@ -25,6 +25,11 @@ public static Color OverlayColor(this Color RGBA2_Bottom, Color RGBA1_Top)
             return Color.FromArgb(255, R, G, B);
         }
 
+        public static Color Reverse(this Color color)
+        {
+            return Color.FromArgb(color.B, color.G, color.R, color.A);
+        }
+
```

Comment:
This bit of logic feels pretty wrong...

Reply:
I actually never use that, it did indeed feel wrong because it was making a new Color just to reverse it, when really I just needed to reverse the write operations for no cost (which I did)

Reply:
Thaaank goodness